### ISSUE TITLE
postinstall script using shelljs doesn't escape $

### ISSUE THREAD

**Issue by patsissons** [ReactiveX/IxJS#163]
**IxJS version:**
`2.3.2`

**Code to reproduce:**
`npm install ix` in windows

**Expected behavior:**
`ix` is installed

**Actual behavior:**
```
> ix@2.3.2 postinstall C:\...\node_modules\ix
> npm i shx && npm run tslib_mjs && npm run tslib_pkg && npm r shx

> ix@2.3.2 tslib_mjs C:\...\node_modules\ix
> shx cp $(node -e "console.log(require.resolve('tslib/tslib.es6.js'))") $(node -e "var r=require,p=r('path');console.log(p.join(p.dirname(r.resolve('tslib')),'tslib.mjs'))")

$(node was unexpected at this time.

C:\...\node_modules\ix>  "C:\...\node_modules\ix\node_modules\.bin\\node.exe"  "C:\...\node_modules\ix\node_modules\.bin\\..\shx\lib\cli.js" cp $(node -e "console.log(require.resolve('tslib/tslib.es6.js'))") $(node -e "var r=require,p=r('path');console.log(p.join(p.dirname(r.resolve('tslib')),'tslib.mjs'))")

npm ERR! Windows_NT 10.0.16299
npm ERR! argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "run" "tslib_mjs"
npm ERR! node v6.11.4
npm ERR! npm  v3.10.10
npm ERR! code ELIFECYCLE
npm ERR! ix@2.3.2 tslib_mjs: `shx cp $(node -e "console.log(require.resolve('tslib/tslib.es6.js'))") $(node -e "var r=require,p=r('path');console.log(p.join(p.dirname(r.resolve('tslib')),'tslib.mjs'))")`
npm ERR! Exit status 255
npm ERR!
npm ERR! Failed at the ix@2.3.2 tslib_mjs script 'shx cp $(node -e "console.log(require.resolve('tslib/tslib.es6.js'))") $(node -e "var r=require,p=r('path');console.log(p.join(p.dirname(r.resolve('tslib')),'tslib.mjs'))")'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the ix package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     shx cp $(node -e "console.log(require.resolve('tslib/tslib.es6.js'))") $(node -e "var r=require,p=r('path');console.log(p.join(p.dirname(r.resolve('tslib')),'tslib.mjs'))")
```

**Additional information:**
Related commit: 4493754d9fbbc061347d8ef785704b61ecb486c5

I'm guessing this is due to `$(node` being interpreted literally rather than as an embedded expression.

**Comment by trxcllnt** [comment]
@patsissons thanks I'll fix and publish a new release asap. I'm not on Windows so I can't test locally (I'll add an `npm install` smoketest in appveyor though), do I just have to escape `$` characters with `\` ?

**Comment by trxcllnt** [comment]
@patsissons 564c60044a693515de253360a2f420ecca392412 reverts 4493754d9fbbc061347d8ef785704b61ecb486c5, and just published 2.3.3 to npm. I guess we'll have to wait until https://github.com/Microsoft/tslib/pull/44 is merged before we can have true `--experimental-modules` support

**Comment by patsissons** [comment]
I'm not actually sure how you would make this work on windows, i was trying to figure something out to shim a fix in, but couldn't really come up with anything reasonable. Thanks for the quick turn around.
